cosmosdb/constants.bal

```diff
@@ -14,21 +14,21 @@
 // specific language governing permissions and limitations
 // under the License. 
 
-# Represents the **Consistency Level Override** for document create and update.
+# The **Consistency Level Override** for document create and update.
```

Comment:
Why is "Consistency Level Override" in title caps and bold? how about including a description that explains what it is?